client/my-sites/stats/stats-empty-state/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+interface Props {
+	stateType: string;
+}
+
+export default function StatsEmptyState( { stateType }: Props ) {
+	const _stateType = String( stateType ).toLowerCase().trim();
```

Comment:
```suggestion
export default function StatsEmptyState( { type }: Props ) {
	const typeForDisplay = String( stateType ).toLowerCase().trim();
```

Nit: Improved property and variable names